raiden/tests/integration/transfer/test_refundtransfer.py

```diff
@@ -62,12 +61,13 @@ def run_test_refund_messages(raiden_chain, token_addresses, deposit, network_wai
     )
 
     # Exhaust the channel App1 <-> App2 (to force the refund transfer)
-    exhaust_amount = calculate_amount_to_drain_channel(deposit)
+    # Here we make a direct transfer, no fees are charged so we should
```

Comment:
nitpick: IMO we should not use the expression `direct transfer` here because it can be confusing. We used to have a different message in the protocol for this, which is not available anymore.

Perhaps we can call it a `1 hop transfer`?